streaming/base/format/base/writer.py

```diff
@@ -68,13 +60,8 @@ def __init__(self,
                  hashes: Optional[List[str]] = None,
                  size_limit: Optional[int] = 1 << 26,
                  extra_bytes_per_shard: int = 0,
-                 extra_bytes_per_sample: int = 0) -> None:
-        if local:
-            pass
-        elif remote:
-            local = mkdtemp()
-        else:
-            raise ValueError('You must provide local and/or remote path(s).')
+                 extra_bytes_per_sample: int = 0,
+                 **kwargs: Any) -> None:
```

Comment:
Dict[str, Any] ?

Reply:
The keyword args is a mapping of parameter name to its value and can be any number of parameters, not necessarily to be a dict. Hence, Any would be a good choice here.